.github/workflows/osx-sccache.yml

```diff
@@ -0,0 +1,52 @@
+name: osx-sccache
+
+on:
+  push:
+    branches: [master, release-*, gha-test-*]
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  max_threads: 16
+  pre_command: cd /opt/git/diem/
+
+jobs:
+  update-sccache:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # This ensures that the tip of the PR is checked out instead of the merge between the base ref and the tip
+          # On `push` this value will be empty and will "do-the-right-thing"
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0 #get all the history!!!
+      - uses: ./.github/actions/check-machine
+        if: ${{ steps.rust-changes.outputs.changes-found == 'true' }}
+        with:
+          webhook-gha-hardware: ${{ secrets.WEBHOOK_GHA_HARDWARE }}
+      - id: changes
+        name: determine changes
+        uses: ./.github/actions/changes
+        with:
+          workflow-file: ci-test.yml
+      - id: rust-changes
+        name: find rust/cargo changes.
+        uses: ./.github/actions/matches
+        with:
+          pattern: '^documentation\|^docker\|^scripts\|^.circleci'
```

Comment:
I think we need to skip the developers website stuff (https://github.com/diem/diem/tree/master/developers.diem.com)

Reply:
the next line has invert set to true.... this is the list of what gets skipped.